### DISCUSSION TITLE
linux的足球游戏太少了，我有一个想法就是把我在steam买的linux正版FM2018共享出来给高手破解一下。

### DISCUSSION THREAD

**Discussion by bodo005** [Gamuxorg/bbs#245] [投递和请求游戏]
游戏名称：Football Manager 2018
游戏名称(英文)：自带中文
下载地址 :（如果有的话）：steam已经下架了，我只传了steam原生的linux游戏文件，还需要高手给破解一下。天翼云地址：

https://cloud.189.cn/web/share?code=NR3Qzqz2Qbyy（访问码：nt7s）

**Comment by hyh19962008** [comment]
感谢分享，我们会尽快处理。目前确实体育和竞速类的游戏非常稀缺，资源网站上也很少，您的分享是很好的补充。

**Comment by hyh19962008** [comment]
已添加。
Steam游戏的破解一般替换目录下的libsteam_api.so就可以了。
https://gitlab.com/Mr_Goldberg/goldberg_emulator

**Comment by kero990** [comment]
其实有一个开源版的足球游戏
https://github.com/vi3itor/GameplayFootball
挺早的项目，编译不难，我在arm64上编译了一份
不过我不会玩，而且显示效果只能说一般。
说实话，还不如去玩ppsspp的足球游戏，psp甚至一直有人在把最新的球员信息导入原版游戏来做成所谓的“2024”版
